po/de.po

```diff
@@ -113,7 +113,7 @@ msgstr "Adresse eingetragen."
 
 #: alias.c:394
 msgid "Error seeking in alias file"
-msgstr "Fehler beim Suchen in alias Datei"
+msgstr "Fehler beim Suchen in Addressbuchdatei"
```

Comment:
I would advocate to differentiate `alias` stuff from addressbook which are inherently two different things. An alias is just a short mnemonic you use for a long address string.

An addressbook on the other hand is an index you lookup someone on, based on a given filed. There applications to manage addressbooks, and mutt provides alias mechanism.

My opinion is that calling aliases addressbook can lead to confusion of what it really is.

But I'm not using a localised neomutt, so whatever feels better for native germans using l10n mutt translation. I won't stand in the way 😉